roles/default/kiali-deploy/tasks/openshift/os-main.yml

```
@@ -26,6 +26,8 @@
     - "templates/openshift/role-controlplane.yaml"
     - "templates/openshift/rolebinding.yaml"
     - "templates/openshift/rolebinding-controlplane.yaml"
+    - "templates/openshift/clusterrole.yaml"
+    - "templates/openshift/clusterrolebinding.yaml"
```

would like these renamed:
```suggestion
    - "templates/openshift/clusterrole-oauth.yaml"
    - "templates/openshift/clusterrolebinding-oauth.yaml"
```